providers/darwin/process_darwin.go

```diff
@@ -221,11 +225,11 @@ func kern_procargs(pid int, p *process) error {
 			break
 		}
 
-		parts := bytes.SplitN(l, []byte{'='}, 2)
-		key := string(parts[0])
+		parts := strings.SplitN(l, "=", 2)
+		key := parts[0]
 		var value string
 		if len(parts) == 2 {
-			value = string(parts[1])
+			value = parts[1]
 		}
```

Comment:
Suggest `strings.Cut` for this.
```
key, val, _ := strings.Cut(l, "=")
env[key] = val
```

Reply:
That's much nicer.

Reply:
The other nice addition, from go1.20 on is https://pkg.go.dev/strings#CutPrefix and https://pkg.go.dev/strings#CutSuffix. I found myself missing them in the okta API package.